package.json

```diff
@@ -33,13 +33,15 @@
     "cookie-parser": "^1.4.3",
     "express": "^4.14.1",
     "helmet": "^3.4.0",
+    "mongodb": "^2.2.24",
     "nhsuk-bunyan-logger": "nhsuk/bunyan-logger#1.0.4",
     "nock": "^9.0.6",
```

Comment:
It isn't clear what line the comment is about. I think it appears as the last line that you can see i.e. `nock`